addons/mail/static/src/views/web/form/mail_template_form.js

```diff
@@ -0,0 +1,26 @@
+/** @odoo-module **/
+
+import { FormController } from "@web/views/form/form_controller";
+import { formView } from "@web/views/form/form_view";
+import { registry } from "@web/core/registry";
+
+const { useEffect } = owl;
+
+class MailTemplateFormController extends FormController {
```

Comment:
I am a bit confused, seems it was doing more than that previously ? Because it seems you are just doing an unlink, don't you ? Then do we need something specific ? As anyway there is a delete action in the "more" menu, not sure to see the final added value. Sorry for iterative thinking ^^

Reply:
`this.deleteRecord` is actually the method used with the gear menu `Delete`, so it adds a confirmation modal to the button in JS